README.md

```diff
@@ -0,0 +1,204 @@
+
+To run the project, execute the following command:
+
+[$ npm start]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+ 👤 **Abubakar Ummar**
+
+- GitHub: [@Haywayaheadshot](https://github.com/Haywayaheadshot)
+- LinkedIn: [@Abubakar Ummar](https://www.linkedin.com/in/abubakar-ummar/)
+
+ 👤 **Abdul Nganje**
+
+- GitHub: [@Nganje](https://github.com/asnganje)
+- Twitter: [@asnganje](https://twitter.com/asnganje)
+- LinkedIn: [@Nganje](https://www.linkedin.com/in/abdulrahman-nganje-a6436935/)
+
+ 👤 **Mohamed Eldeeb**
+
+- GitHub: [@Mohammed](https://github.com/eng-mohamed-eldeeb)
+- Twitter: [@El_Deeb](https://twitter.com/eldeeb_3o)
+- LinkedIn: [@Mohammed Eldeeb](https://www.linkedin.com/in/eng-mohamed-eldeeb/)
+
+👤 **Kelvin Benjamin**
+
+- GitHub: [@Kelvin-Ben](https://github.com/Kelvin-Ben)
+- Twitter: [@kevoyout144](https://twitter.com/kevoyout144)
+- LinkedIn: [@Kelvin-Ben](https://www.linkedin.com/in/kelvin-ben-323043173/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+> We will create a React and Ruby on Rails application that allows the user to: 
+
+- [ ] **Register and login, so that the data is private to them**
+- [ ] **User is able to book an apointmwnt with a specific doctor**
+- [ ] **Able to use other services**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Anyone who finds this project interesting is welcome to give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse for the opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[How do i run the project in my local machine?]**
+
+  - [You need to clone the project in your local environment through the terminal]
+
+- **[What database are you using for this project?]**
+
+  - [POSTGRESQL]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
[ ] This is just a placeholder for you to create and include a license file. Kindly remove this note, as it's meant just for you. 👍